app/static/js/settings.js

```diff
@@ -0,0 +1,24 @@
+const tabBtn =
```

Comment:
This, and the for in the last lines of this file, should only run after you ensure the page is fully loaded (window -> load or document's DOMContentLoaded)